MAINTAINING.md

```diff
@@ -4,8 +4,9 @@
 
 ### 1. Update the version number.
 
-- Update `version` in [package.json in `frontend/`](https://github.com/yorkie-team/codepair/blob/1f10dff1d9c253f921ba136c812383fbb292078f/frontend/package.json#L5) and [package.json in `backend/`](https://github.com/yorkie-team/codepair/blob/1f10dff1d9c253f921ba136c812383fbb292078f/backend/package.json#L3).
-- Update `version` in `package-lock.json` in `frontend/` and `backend/`. It will be automatically updated when you run `npm install`.
+- For packages that are independently versioned (e.g., `frontend/`, `backend/`), update the `version` field in their respective `package.json` files when making a release.
+- Otherwise, centralize version management at the root `package.json` if using a unified versioning strategy.
```

Comment:
CodePair uses global version management.
Can we manage the version only in `package.json`(not `frontend/package.json`, `backend/package.json`)?

When I write the version only in `package.json`, it doesn't detect `import.meta.env.PACKAGE_VERSION`.

It is just a question :)

Reply:
As we don’t have to release each package separately like other monorepos, it should be okay to manage the version in the root package.json.